src/components/nav-icon-item/nav-icon-item.stories.tsx

```diff
@@ -119,8 +147,9 @@ export const ReactAnchorUsage: Story = {
   render: (args) => {
     return (
       <FlexContainer>
```

Comment:
> `NavIconItem` requires `aria-label` to be provided, but none of these stories are providing a value for that prop

That's because I've passed the `aria-label` prop in the meta variable, so each story only needs to provide the remaining required API props